app/views/shared/provider/_header.html.slim

```diff
@@ -1,15 +1,32 @@
 = javascript_pack_tag 'navigation'
 = javascript_pack_tag 'PF4Styles/header'
+= javascript_pack_tag 'CrossConsole'
 
-header.pf-c-page__header id="user_widget" class='Header' class=('Header--master' if current_account.master?) role='banner'
+header.pf-c-page__header id="user_widget" class='app-page__header pf-m-no-toggle' class=('Header--master' if current_account.master?) role='banner'
 
-  = link_to provider_admin_dashboard_path, class: 'pf-c-page__header-brand Header-link', title: 'Dashboard' do
+  = link_to provider_admin_dashboard_path, class: 'pf-c-page__header-brand-link Header-link', title: 'Dashboard' do
       .Header-logo class=('Header-logo--withIcon' if active_menu?(:main_menu, :dashboard))
         span Red Hat 3scale API Management
+        img[src="./../../../../assets/images/Logo-Red_Hat-Integration-A-Reverse-RGB.png"]
```

Comment:
we are currently using encoded SVG embedded in css: https://github.com/3scale/porta/blob/master/app/assets/stylesheets/provider/_header.scss#L17